FEXCore/Scripts/config_generator.py

```diff
@@ -217,6 +217,20 @@ def print_man_environment_tail():
     ],
     "''", True)
 
+    print_man_env_option(
+    "FEX_PORTABLE",
+    [
+    "Forces FEX in to a more portable installation",
+    "This does a couple of things:",
+    "- Skips executing binfmt_misc handlers through execve",
+    "- global config (typically installed in /usr/share/fex-emu/) disabled",
+    "- FEX_APP_CONFIG_LOCATION defaults to FEXInterpreter relative paths instead of {$HOME, $XDG_DATA_HOME}/.fex-emu/",
```

Comment:
Should `FEX_APP_CONFIG_LOCATION` and `FEX_APP_DATA_LOCATION` get their own sections too? Not sure what the typical conventions are for man pages here.

Reply:
Those already have their own sections above this one.